website/src/app/kb/deploy/resources/readme.mdx

```diff
@@ -10,11 +17,12 @@ are expected to be reachable by all gateways in the same site.
 
 DNS-based resources match the address entered and all subdomains.
```

Comment:
```suggestion
DNS-based resources match the address entered and all subdomains.

e.g. `github.com` will match `imgs.github.com` and `github.com` itself, including all SSH and HTTPS traffic.
```

Reply:
Updated this section to be much more detailed and provide examples